Assets/Mirror/Runtime/Transport/TelepathyTransport.cs

```diff
@@ -117,8 +126,30 @@ public void LateUpdate()
             // note: we need to check enabled in case we set it to false
             // when LateUpdate already started.
             // (https://github.com/vis2k/Mirror/pull/379)
-            while (enabled && ProcessClientMessage()) { }
-            while (enabled && ProcessServerMessage()) { }
+            if (!enabled)
+                return;
+
+            // process a maximum amount of client messages per tick
+            for (int i = 0; i < clientMaxReceivesPerTick; ++i)
+            {
+                // stop when there is no more message
+                if (!ProcessClientMessage())
+                {
+                    //if (i > 0) Debug.Log("Telepathy Client processed " + i + " messages this tick.");
```

Comment:
```suggestion

```
Don't leave commented out code. If you want to keep the message then we can use the new logger class for it.